src/map.h

```diff
@@ -578,50 +581,54 @@ void draw_rough_circle(std::string type, int x, int y, int rad);
- /** Makes spores at the respective x and y. source is used for kill counting */
- void create_spores( const tripoint &p, Creature* source = nullptr );
- /** Checks if a square should collapse, returns the X for the one_in(X) collapse chance */
- int collapse_check(const int x, const int y);
- /** Causes a collapse at (x, y), such as from destroying a wall */
- void collapse_at(const int x, const int y);
- /** Returns a pair where first is whether something was smashed and second is if it was a success */
- std::pair<bool, bool> bash(const int x, const int y, const int str, bool silent = false,
-                            bool destroy = false, vehicle *bashing_vehicle = nullptr);
- // spawn items from the list, see map_bash_item_drop
- void spawn_item_list(const std::vector<map_bash_item_drop> &items, int x, int y);
- /** Keeps bashing a square until it can't be bashed anymore */
- void destroy(const int x, const int y, const bool silent = false);
+void add_corpse( const tripoint &p );
+
+
+// Terrain changing functions
+    void translate( const ter_id from, const ter_id to ); // Change all instances of $from->$to
+    void translate_radius( const ter_id from, const ter_id to, const float radi, const tripoint &p );
+    bool close_door( const tripoint &p, const bool inside, const bool check_only );
+    bool open_door( const tripoint &p, const bool inside, const bool check_only = false );
+// Destruction
+     /** Keeps bashing a square until it can't be bashed anymore */
     void destroy( const tripoint &p, const bool silent = false);
- /** Keeps bashing a square until there is no more furniture */
- void destroy_furn(const int x, const int y, const bool silent = false);
- void crush(const int x, const int y);
- void shoot(const int x, const int y, int &dam, const bool hit_items,
-            const std::set<std::string>& ammo_effects);
+    void destroy( int, int ) = delete;
+    /** Keeps bashing a square until there is no more furniture */
+    void destroy_furn( const tripoint &p, const bool silent = false );
+    void crush( const tripoint &p );
+    void shoot( const tripoint &p, int &dam, const bool hit_items,
+                const std::set<std::string>& ammo_effects );
+    /** Checks if a square should collapse, returns the X for the one_in(X) collapse chance */
+    int collapse_check( const tripoint &p );
+    /** Causes a collapse at (x, y), such as from destroying a wall */
+    void collapse_at( const tripoint &p );
+    /** Returns a pair where first is whether something was smashed and second is if it was a success */
+    std::pair<bool, bool> bash( const tripoint &p, const int str, bool silent = false,
+                                bool destroy = false, vehicle *bashing_vehicle = nullptr );
+    std::pair<bool, bool> bash( int, int, const int, bool silent = false,
+                                bool destroy = false, vehicle *bashing_vehicle = nullptr ) = delete;
```

Comment:
C++ is still something I am learning the more I contribute, so I have a question.
I read the open standards draft regarding `= delete` when I saw this here, but want to make sure I got this right.
In this context, you are disallowing use of the old 2D coordinate versions of the functions? Or is it so it does not incorrectly copy the contents over to tripoint type? (I know it can be used to disallow overloaded conversions, or some jazz).